File: Level1/최대공약수와 최소공배수.py
def gcd(x, y):
    while y > 0:
        x, y = y, x % y
    return x

def lcm(x, y):
    return x * y // gcd(x, y)

def solution(n, m):
    answer = [gcd(n, m), lcm(n, m)]
    return answer

File: Level1/최대공약수와 최소공배수.java
class Solution {
    public int gcd(int x, int y) {
        if (x % y == 0) {
            return y;
        }
        return gcd(y, x % y);
    }
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        answer[0] = gcd(n, m);
        answer[1] = n * m / answer[0];
        return answer;
    }
    
}